src/Connection.php

```diff
@@ -31,6 +31,22 @@ class Connection
      */
     public $driverType;
 
+    /**
+     * Stores the driverType => connectionClass array for resolving.
```

Comment:
we should probably call driverType driver scheme (it is scheme when specified by URI, do more research on this how it is called internally in PHP/PDO lib)